.gitignore

```diff
@@ -1,42 +1,40 @@
-Gemfile.lock
-Gemfile.local
-coverage
 *.gem
+.attribute.yml
```

Comment:
These changes seem unrelated to the purpose of the PR. Could you clean it up a bit?